armi/reactor/converters/geometryConverters.py

```diff
@@ -892,14 +905,15 @@ def _getHomogenizedBlockType(self, numHexBlockByType):
         # Find the most common block type out of the types in the block mixture type exclusions list
         excludedBlockTypesInBlock = set(
             [
-                x
+                Flags.toString(x)
```

Comment:
explain in a comment why we're trafficking in strings rather than flags in this section